src/routes/console/project-[project]/auth/user-[user]/header.svelte

```diff
@@ -39,12 +38,7 @@
         <CoverTitle href={`/console/project-${projectId}/auth`}>
             {$user.name ? $user.name : '-'}
         </CoverTitle>
-        <Copy value={$user.$id} event="user">
-            <Pill button>
-                <span class="icon-duplicate" aria-hidden="true" />
-                User ID
-            </Pill>
-        </Copy>
+        <ID value={$user.$id} event="user">{$user.$id}</ID>
```

Comment:
Why we have `x?.$id` in some places but not here?

Reply:
not sure, but the data should always be here 👍🏻